hello.go

```diff
@@ -157,7 +115,7 @@ func main() {
 		Service:  service,
 		Revision: revision,
 		Project:  project,
-		Region:   region,
+		Zone:     zone,
```

Comment:
blocker: Cloud Run uses region not zone. While we can extract the region from the zone, we can't use it directly. We should teach using region and how to extract region from zone.

Reply:
Ok, thanks for this relevant comment! 
I think `region` is missing from `metadata` package.
I need to check (sorry for delay) here: https://github.com/googleapis/google-cloud-go/blob/main/compute/metadata/metadata.go